src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandlerTest.java

```diff
@@ -35,110 +56,177 @@ public class AdjournCaseAboutToSubmitHandlerTest {
+        + "and no directions are being made, then should send a new hearing request in hearings API")
     @Test
-    public void givenValidSubmissionWithGeneratedDateSet_thenSetGeneratedDateToNowAndDoNotDisplayAnError() {
+    public void givenCaseCannotBeListedRightAwayAndNoDirectionsBeingMade_thenNewHearingRequestSent() {
+        PreSubmitCallbackResponse<SscsCaseData> response = cannotBeListedAndNoDirectionsGiven();
 
-        callback.getCaseDetails().getCaseData().setAdjournCaseGeneratedDate("2018-01-01");
+        assertHearingCreated(response);
+    }
 
-        handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
+    @DisplayName("When adjournment is enabled and case is LA and case can be listed right away "
+        + "then should send a new hearing request in hearings API")
+    @Test
+    public void givenCanBeListedRightAway_thenNewHearingRequestSent() {
+        PreSubmitCallbackResponse<SscsCaseData> response = canBeListed();
 
-        assertEquals(LocalDate.now().toString(), callback.getCaseDetails().getCaseData().getAdjournCaseGeneratedDate());
+        assertHearingCreated(response);
     }
-    
+
+    @DisplayName("When adjournment is enabled and case is LA and case cannot be listed right away "
+        + "and directions are being made, then should not send any messages")
     @Test
-    @Parameters({"ABOUT_TO_START", "MID_EVENT", "SUBMITTED"})
-    public void givenANonCallbackType_thenReturnFalse(CallbackType callbackType) {
-        assertFalse(handler.canHandle(callbackType, callback));
+    public void givenCaseCannotBeListedRightAwayAndDirectionsAreBeingMade_thenNoMessagesSent() {
+        PreSubmitCallbackResponse<SscsCaseData> response = cannotBeListedAndDirectionsGiven();
+
+        verifyNoInteractions(hearingMessageHelper);
+
+        assertThat(response.getErrors()).isEmpty();
     }
 
-    @Test(expected = IllegalStateException.class)
-    public void throwsExceptionIfItCannotHandleTheAppeal() {
-        when(callback.getEvent()).thenReturn(EventType.APPEAL_RECEIVED);
-        handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
+    private PreSubmitCallbackResponse<SscsCaseData> cannotBeListedAndNoDirectionsGiven() {
+        return getResponseWithYesNoCanBeListedAndYesNoDirections(NO, NO);
+    }
+
+    private void assertHearingCreated(PreSubmitCallbackResponse<SscsCaseData> response) {
+        verify(hearingMessageHelper, times(1))
+            .sendListAssistCreateHearingMessage(sscsCaseData.getCcdCaseId());
+
+        assertThat(response.getErrors()).isEmpty();
+    }
+
+    private PreSubmitCallbackResponse<SscsCaseData> canBeListed() {
+        return getResponseWithYesNoCanBeListedAndYesNoDirections(YES, NO);
+    }
+
+    private PreSubmitCallbackResponse<SscsCaseData> cannotBeListedAndDirectionsGiven() {
+        return getResponseWithYesNoCanBeListedAndYesNoDirections(NO, YES);
+    }
```

Comment:
I think this is too much abstracting for a test, just call getResponseWithYesNoCanBeListedAndYesNoDirections(NO, YES) in the test itself. Remember you don't want to make your tests too complex as it might get to the point you need a test to test your test and it can make understanding your test harder for future devs.